spec/models/claims/claim_spec.rb

```diff
@@ -110,4 +110,41 @@
       end
     end
   end
+
+  describe "#ready_to_be_checked?" do
+    it "returns true if the claim is ready to be checked" do
```

Comment:
This spec description doesn't provide the context of the intended behaviour. What does "ready to be checked" actually mean in terms of behaviour?

```suggestion
    it "returns true if the claim's mentors all have their hours recorded" do
```